src/components/AppButton.js

```diff
@@ -0,0 +1,11 @@
+export const AppButton = (props) => {
+  const { url, children, type } = props;
+  const className = "button button-" + type;
+  return (
+    <>
```

Comment:
Better to use Fragment only when you return more components. Here you return only one

Reply:
Ok.